monitorable/config/usecase/getconfig.go

```diff
@@ -56,12 +66,62 @@ func (cu *configUsecase) GetConfig(params *models.ConfigParams) *models.ConfigBa
 				field = subMatch[0][1]
 			}
 
+			var configField = structs.Fields(models.Config{})
```

Comment:
To be consistent with the rest of the code, replace the var xxx = aaa by xxx := aaa